pocs/scheduler/constraint.py

```diff
@@ -177,11 +177,12 @@ def get_score(self, time, observer, observation, **kwargs):
         veto = False
         score = self._score
 
-        target = observation
+        target = observation.field
         observed_list = kwargs.get('observed_list', observer.observed_list)
 
-        if target in observed_list:
-            veto = True
+        for seq_time, Observation in observed_list:
+            if target = observation:
```

Comment:
You are assigning (`=`) here instead of comparing (`==`). Which might not show an error.